flink-tests/src/test/java/org/apache/flink/test/recovery/AbstractTaskManagerProcessFailureRecoveryTest.java

```diff
@@ -258,7 +258,7 @@ protected static void printProcessLog(String processName, TestProcess process) {
 			System.out.println("-----------------------------------------");
 			System.out.println(" BEGIN SPAWNED PROCESS LOG FOR " + processName);
 			System.out.println("-----------------------------------------");
-			System.out.println(process.getOutput().toString());
+			System.out.println(process.getErrorOutput().toString());
```

Comment:
Why is this change ok? Is the log being logged to stderr?

Reply:
I suppose this test initially needed the stderr in case of error.
`TestProcess#getOutput` function name was confusing as it actually contained `Process#getErrorStream` of the underlying process in `TestProcessBuilder#start`. Now I also needed to expose the normal stdout so I decided to change the naming.